tailwind.config.js

```diff
@@ -0,0 +1,17 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    "./public/*.html",
+    "./app/assets/**/*",
```

Comment:
Watching `assets` dir is probably not necessary, unless you plan to write some HTML-in-JS. Probably does not hurt too, Tailwind is pretty fast after all.

Reply:
Very good point! I've certainly no intentions to do this right now, so I'll remove this until if/when it becomes necessary :)